packages/serverless-components/nextjs-component/src/component.ts

```diff
@@ -205,7 +205,9 @@ class NextjsComponent extends Component {
     const buildConfig: BuildOptions = {
       enabled: inputs.build
         ? // @ts-ignore
-          inputs.build !== false && inputs.build.enabled !== false
+          inputs.build !== false && // @ts-ignore
```

Comment:
Guess we should figure out how to not have to use `@ts-ignore`, though not really due to your PR